kytos/utils/config.py

```diff
@@ -52,6 +62,27 @@ def __init__(self, config_file='~/.kytosrc'):
                 os.chmod(self.config_file, 0o0600)
                 self.config.write(output_file)
 
+    @staticmethod
+    def _create_data_files_directory():
+        """Install data_files in the /etc directory."""
+        parent_dir = Path(os.path.abspath(os.path.dirname(__file__))).parent
+
+        etc_kytos = os.path.join(BASE_ENV, ETC_KYTOS)
+
+        if not os.path.exists(etc_kytos):
+            os.makedirs(etc_kytos)
+
+        src = os.path.join(parent_dir, KYTOS_SKEL_PATH)
+        skel = KYTOS_SKEL_PATH.replace('templates/', '')
+        dst = os.path.join(BASE_ENV, ETC_KYTOS, skel)
+        if os.path.exists(dst):
+            if not os.listdir(dst):
+                # Path already exists but it's empty, so we'll populate it
+                # We remove it first to avoid an exception from copytree
+                os.rmdir(dst)
+        else:
+            shutil.copytree(src, dst)
```

Comment:
Wrong logic here: if `dst` exists but is empty, only `rmdir` will run, without the `copytree`.